src/main/java/com/example/event_manager/service/PersonServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.example.event_manager.service;
+
+import com.example.event_manager.model.Person;
+
+import java.util.List;
+
+public class PersonServiceImpl implements PersonService {
+  @Override
+  public boolean save(Person personForm) {
+    return false;
+  }
+
+  @Override
+  public void delete(Long id) {
+  }
+
+  @Override
+  public List<Person> findAll() {
+    return null;
+  }
+
+  @Override
+  public Person findById(Long id) {
+    return null;
```

Comment:
why it returns null?

Reply:
if it throws then thow this exception in method implementation - in other case it make no sense to add throws to signature